based_func_alicat.py

```diff
@@ -0,0 +1,85 @@
+#code to push
+
+
```

Comment:
Can we move this file into `src/tomato_alicat/...`? Then it will be a part of the package...